modules/govuk_jenkins/manifests/jobs/ask_export.pp

```diff
@@ -69,14 +69,17 @@
     notify  => Exec['jenkins_jobs_update'];
   }
 
-  if $run_daily {
-    @@icinga::passive_check { "ask_export_${::hostname}":
-        ensure              => present,
-        service_description => $service_description,
-        host_name           => $::fqdn,
-        freshness_threshold => 86400, # 24 Hours
-        action_url          => "https://${deploy_jenkins_domain}/job/ask-export/",
-        # notes_url           => monitoring_docs_url(ask-export);
-    }
+  $freshness_threshold = $run_daily ? {
+    true  => 86400, # 24 hours
+    false => '',
+  }
+
+  @@icinga::passive_check { "ask_export_${::hostname}":
+      ensure              => present,
+      service_description => $service_description,
+      host_name           => $::fqdn,
+      freshness_threshold => $freshness_threshold,
+      action_url          => "https://${deploy_jenkins_domain}/job/ask-export/",
+      # notes_url           => monitoring_docs_url(ask-export);
```

Comment:
If i can get https://github.com/alphagov/govuk-developer-docs/pull/3348 approved we can uncomment this :)

Reply:
👀

Reply:
:D Thanks!

Reply:
I've merged that now so if you could uncomment this we're all good! :)

Reply:
Great just pushed up a commit to add that